UPGRADE.md

```diff
@@ -28,6 +28,8 @@
 
 ### Grid / GridBundle
 
+* Add ``$formOptions`` array to the ``Sylius\Component\Grid\Definition\Filter`` model where custom options for filter form types could be defined instead of defining it in ``$options``.
```

Comment:
In the UPGRADE file we should explain how to upgrade the app = how to update grid configurations to work after these changes. :)